docs/concepts/our-tech-stack.md

```diff
@@ -1,4 +1,9 @@
-## Why we choose our tech stack
+# Our Tech Stack
+Leverage was built around the [AWS Well Architected Framework](https://aws.amazon.com/architecture/well-architected/) and it uses a stack that includes [Terraform](https://www.terraform.io/), [Ansible](https://www.ansible.com/), [Helm](https://helm.sh/) and other tools.
+
+We are also adopters and supporters of Kubernetes and the Cloud Native movement, which you should become self-evident as you keep exploring our technology stack.
```

Comment:
```suggestion
We are also adopters and supporters of Kubernetes and the Cloud Native movement, which should become self-evident as you keep exploring our technology stack.
```

Reply:
This one was already suggested by @angelofenoglio and should be fixed.